### PR TITLE
use new JSON parser for tokens

### PR DESCRIPTION
**Pull request by jesuspg**
@hmunfru @jframos

### CODE REVIEW

**Comment by hmunfru** [comment]
LGTM

File: rest-api/src/main/java/com/telefonica/euro_iaas/sdc/rest/auth/OpenStackAuthenticationProvider.java [hunk 1]
```diff
@@ -213,32 +212,36 @@ public PaasManagerUser authenticationFiware(String token, String tenantId) {
...
-        if (!tenantId.equals(authenticateResponse.getToken().getTenant().getId())) {
-            throw new AuthenticationServiceException("Token " + authenticateResponse.getToken().getTenant().getId()
-                    + " not valid for the tenantId provided:" + tenantId);
-        }
+            Set<GrantedAuthority> authsSet = new HashSet<GrantedAuthority>();
 
-        Set<GrantedAuthority> authsSet = new HashSet<GrantedAuthority>();
+            PaasManagerUser user = new PaasManagerUser(responseUserName, token, authsSet);
 
-        if (authenticateResponse.getUser().getRoles() != null) {
```
Comment (flopezag) [inline rest-api/src/main/java/com/telefonica/euro_iaas/sdc/rest/auth/OpenStackAuthenticationProvider.java]: Why we do not need to get the roles?

Comment (jesuspg) [inline rest-api/src/main/java/com/telefonica/euro_iaas/sdc/rest/auth/OpenStackAuthenticationProvider.java]: because SDC never use it

Comment (flopezag) [inline rest-api/src/main/java/com/telefonica/euro_iaas/sdc/rest/auth/OpenStackAuthenticationProvider.java]: NTC

**Comment by flopezag** [comment]
I asume that the reason to delete the keystone schema files is due to you want to use only json message, so it is not necessary to have them. It is correct?

**Comment by jesuspg** [comment]
yes

**Comment by flopezag** [comment]
LGTM

**Pull request merged**